explainer.md

```diff
@@ -0,0 +1,107 @@
+A previous iteration of this spec included both Screen and System wake locks.
+The latter has been moved to a separate spec, and is out of scope for this
+explainer.
+
+## API
+The API surface is not big: users can request a screen wake lock; if the
+request succeeds it can be either manually released later or released
+automatically by the platform due to OS-specific policies that might be in
+place. The code looks like this:
+
+```js
+const lock = await navigator.wakeLock.request("screen");
+
+lock.addEventListener("release", () => {
+  console.log(`Has the lock been released? ${lock.released}`);
+});
+await lock.release();
+```
+
+### WakeLock
+The `WakeLock` object is part of `Navigator`. It has a single method,
+`request(type)`. The `type` argument is of the `WakeLockType` enum type, which
+right now has a single value, _"screen"_.
+
+`WakeLock.request()` performs permission checks, verifies that the document
+issuing the request is visible and, if all checks succeed, asynchronously
+requests a lock from the underlying platform to prevent the screen from turning
+off.
+
+A successful `WakeLock.request()` invication returns a promise that will
+ultimately resolve with a `WakeLockSentinel` object.
+
+### WakeLockSentinel
+A `WakeLockSentinel` object provides a handle to the lock that has been
+acquired.
+
+`WakeLockSentinel` has two read-only attributes for introspection: `released`
+indicates whether the lock has already been released, `type` returns the lock's
+type (it is always _"screen"_ at the moment).
+
+The `WakeLockSentinel.release()` method is used to release the platform wake
+lock. It asynchronously emits a _"release"_ event that users can listen for in
+order to know that the lock is no longer being held.
+
+The lock might also be released automatically by the platform without
+`WakeLockSentinel.release()` being called (for example, when the page is no
+longer visible). The _"release_" event is still emitted the same way.
+
+## Security considerations
+* To avoid possible user fingerprinting issues, `WakeLock.request()` does not
+  indicate to API users whether the actual call to obtain a lock from the
+  operating system has succeeded or not. In other words, successful
+  `WakeLock.request()` calls must be treated by users as **advisory-only**.
+
+* Screen Wake Locks can only be acquired and held if the document is visible.
+  If `WakeLock.request()` is called while the document is hidden, the request
+  will be denied with a `NotAllowedError`. If the page is hidden after a lock
+  has been acquired, it will be released automatically.
+
+* Application of a wake lock causes various device components such as display
+  or CPU to operate at higher power levels than they otherwise would. This can
+  lead to undesirable and potentially dangerous effects such as excessive
+  heating and faster than normal battery charge depletion. The latter is
+  particularly relevant to mobile devices which may not have a stationary power
```

Comment:
I think say "and potentially dangerous" is going a bit overboard, tbh. Browsers are just tapping into publicly available OS APIs to provide this functionality (not arbitrary hacks that could cause a screen or device to catch on fire) - and those are hardly dangerous.  

```suggestion
  or CPU to operate at higher power levels than they otherwise would. This can
  lead to undesirable effects such as faster than normal battery charge depletion. 
  This is particularly relevant to mobile devices, which may not have a stationary power
```

Reply:
Heh this was copied verbatim from the Security and privacy considerations section in the spec. I've sent #287 to reword it there too.